pkg/mev/bloxroute_submit_bundle.go

```diff
@@ -14,69 +14,76 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
-type Builder string
+type BlxrBuilder string
 
 const (
-	BuilderBloxroute    Builder = "bloxroute"
-	BuilderFlashbot     Builder = "flashbots"
-	BuilderBeaverBuild  Builder = "beaverbuild"
-	BuilderRsyncBuilder Builder = "rsync-builder"
-	BuilderAll          Builder = "all"
+	BuilderBloxroute    BlxrBuilder = "bloxroute"
```

Comment:
can enum by number and generate string?

Reply:
Using generated is way too complicated in this case. The BlxrBuilder type is just there to let the user know which one is available.